docs/section4/4-6.md

```diff
@@ -0,0 +1,31 @@
+# Airside Systems – Rule 4-6
+
+**Rule ID:** 4-6
+**Rule Description:** HVAC fans in the proposed design model shall remain on during unoccupied hours in systems primarily serving computer rooms.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules exception #3 for the proposed building  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem 
+**Applicability Checks:** 
+
+1. Applies to hvac systems primarily serving computer rooms.
+
+**Function Call:** 
+
+1. get_HVAC_systems_primarily_serving_comp_rooms()
+
+**Manual Check:** None  
+ 
+**Rule Logic:**  
+**Applicability Check 1:** 
+- Get list of hvac systems that primary serve computer rooms: `hvac_systems_primarily_serving_comp_rooms_list_p = get_HVAC_systems_primarily_serving_comp_rooms (P_RMR)`
+- For each hvac_p in hvac_systems_primarily_serving_comp_rooms_list_p: `for hvac_p in hvac_systems_primarily_serving_comp_rooms_list_p:`
+    - Get operation during unoccupied periods: `operation_during_unoccupied_p = hvac_p.fan_systems[0].operation_during_unoccupied`  
+    - Get minimum_outdoor_airflow: `minimum_outdoor_airflow_p = hvac_p.fan_systems[0].minimum_outdoor_airflow`  
+    **Rule Assertion:**
+    - Case 1: For each applicable hvac system, if fan system operation during unoccupied periods is continuous in the P_RMR: `if operation_during_unoccupied_p == “CONTINUOUS” AND minimum_outdoor_airflow_p > 0: PASS`
+    - Case 2: Else, for each applicable hvac system system: `Else: FAIL and raise_message "THE HVAC SYSTEM SERVES ZONE(S) THAT INCLUDE COMPUTER ROOMS AND THEREFORE MAY WARRANT CONTINUOUS OPERATION DURING UNOCCUPIED HOURS (IF THE HVAC SYSTEMS PROVIDE OA CFM) PER SECTION G3.1-4 SCHEDULES EXCEPTION #3 FOR THE PROPOSED BUILDING. VERIFY WITH A MANUAL CHECK." `
```

Comment:
1. The message still implies a manual check (based on prior UNDERTERMINED outcome). I would consider removing the message.
2. We should either update applicability check to include only systems with minimum_outdoor_airflow_p > 0, or add Case 3 to show PASS if operation_during_unoccupied_p == “INTERMITTENT” AND minimum_outdoor_airflow_p = 0

Reply:
I implemented #2 to make it more robust.